### DISCUSSION TITLE
about contribution

### DISCUSSION THREAD

**Discussion by Sorrowful-free** [cocos/cocos-engine#7519] [Ideas] [answered]
Hello Guys! Could you mark by label your issues like in [google flutter repo](https://github.com/flutter/flutter/issues) it makes contribution easy and effective?

**Comment by jareguo** [comment, marked as answer]
Hi, we will triage our issues later and assign labels more detailed. We really haven't cleaned up the issue very carefully yet, sorry about that. But you are still welcome to submit any issue or PR.
